spec/game_rules_spec.rb

```diff
@@ -0,0 +1,93 @@
+require_relative '../lib/game_rules'
+require_relative '../lib/token'
+
+describe 'Game Rules' do
+    attr_accessor :board
+    include GameRules
+    include Token
+
+    context 'win? 2x2' do
```

Comment:
Why test the 2x2 boards if you have:

```
def board_size_valid?(input:)
        int = input.to_i
        int == 3 || int == 4
end
```

Reply:
you're right! unnecessary and should be testing 4x4 instead.